libs/core/iterator_support/include/hpx/iterator_support/traits/is_range.hpp

```diff
@@ -53,4 +53,7 @@ namespace hpx { namespace traits {
         typedef typename util::detail::iterator<R>::type iterator_type;
         typedef typename util::detail::sentinel<R>::type sentinel_type;
     };
+
+    template <typename T, typename Enable = void>
```

Comment:
We don't need the `typename Enable = void` on the using directive. Nobody is going to pass it directly and we can't specialize the using anyways.